ci/rpm/build.sh

```diff
@@ -15,7 +15,7 @@ set -ex
 mydir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
 ci_envs="$mydir/../parse_ci_envs.sh"
 if [ -e "${ci_envs}" ]; then
-  # shellcheck disable=SC1091
+  # shellcheck disable=SC1091,SC1090
```

Comment:
We might need a subsequent enable= to avoid ignoring it for the rest of the file.  Have you checked this?

Reply:
Running a quick test locally like this:
# shellcheck disable=SC1091,SC1090,SC2086
source ${ci_envs}
source ${ci_envs}

shellcheck seems to complain about the second one, not the first. Looks like it only disable the check for the immediate cmd after.

Reply:
yikes, not sure why it end up with crazy formatting above

Reply:
Shellcheck ignores generally only apply to the command just following them.
The disable needed for a source command varies with the version of shellcheck, which is why we need to disable both codes.

Reply:
thanks for checking.  That differs from pylint